src/Atmos/Model/ref_state.jl

```diff
@@ -191,3 +242,20 @@ function ∇reference_pressure(::ReferenceState, state_auxiliary, grid)
     grad_dg(∇p, gradQ, nothing, FT(0))
     return ∇p
 end
+
+"""
+ρᵢ  = (pᵢ₋₁ - pᵢ - g ρᵢ₋₁ Δzᵢ₋₁/2) / (g  Δzᵢ/2)
```

Comment:
Better docstring here would be nice. Or at least just make it consistent with the docstrings used throughout the rest of the Clima code.

Reply:
I repleced this docstring with a comment since this isn't a user facing function.